Template "node-todo" fails with an assertion error

When executing `npm create @oracle/database-app -- --name 'my-todo' --template 'node-todo'` in an empty directory, I run into this error:

```
gvenzl@gvenzl-mac my-app % npm create @oracle/database-app -- --name 'my-todo' --template 'node-todo'
? Which database connection type would you like to choose? Basic Connection (Protocol, Hostname, Port, Service Name / SID)
? What is your database protocol? tcp
? What is your database hostname? localhost
? What is your database port? 1521
? Which service type would you like to use? Service name
? Please input your database service name:  freepdb1
? What's your database username? gerald
? What's your database password? *********
Generating database app...
✖ An error occured while running database-app#writing
node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

AssertionError [ERR_ASSERTION]: Trying to copy from a source that does not exist: /Users/gvenzl/.npm/_npx/a063f1c0678460cf/node_modules/@oracle/create-database-app/templates/node-todo
    at exports.copy (/Users/gvenzl/.npm/_npx/a063f1c0678460cf/node_modules/@oracle/create-database-app/node_modules/mem-fs-editor/lib/actions/copy.js:67:3)
    at module.exports.copyTpl (/Users/gvenzl/.npm/_npx/a063f1c0678460cf/node_modules/@oracle/create-database-app/node_modules/mem-fs-editor/lib/actions/copy-tpl.js:22:8)
    at default.writing (file:///Users/gvenzl/.npm/_npx/a063f1c0678460cf/node_modules/@oracle/create-database-app/dist/generators/index.js:80:17)
    at Object.<anonymous> (/Users/gvenzl/.npm/_npx/a063f1c0678460cf/node_modules/@oracle/create-database-app/node_modules/yeoman-generator/lib/index.js:1096:23)
    at /Users/gvenzl/.npm/_npx/a063f1c0678460cf/node_modules/@oracle/create-database-app/node_modules/run-async/index.js:49:25
    at new Promise (<anonymous>)
    at /Users/gvenzl/.npm/_npx/a063f1c0678460cf/node_modules/@oracle/create-database-app/node_modules/run-async/index.js:26:19
    at /Users/gvenzl/.npm/_npx/a063f1c0678460cf/node_modules/@oracle/create-database-app/node_modules/yeoman-generator/lib/index.js:1097:9
    at new Promise (<anonymous>)
    at default.executeTask (/Users/gvenzl/.npm/_npx/a063f1c0678460cf/node_modules/@oracle/create-database-app/node_modules/yeoman-generator/lib/index.js:1068:12) {
  generatedMessage: false,
  code: 'ERR_ASSERTION',
  actual: false,
  expected: true,
  operator: '=='
}

Node.js v18.12.1
npm ERR! code 1
npm ERR! path /Users/gvenzl/Downloads/my-app
npm ERR! command failed
npm ERR! command sh -c create-database-app --name my-todo --template node-todo

npm ERR! A complete log of this run can be found in: /Users/gvenzl/.npm/_logs/2024-07-10T19_25_20_146Z-debug-0.log
gvenzl@gvenzl-mac my-app %
```

--- a/README.md
+++ b/README.md
@@ -58,7 +58,7 @@ using the the [`oracledb` database driver](https://github.com/oracle/node-oracle
     [React](https://react.dev/). It is built by [`vite`](https://vitejs.dev/)
 - `node-vue`: A starter template that uses Node.js and
     [Vue.js](https://vuejs.org/). It is built by [`vite`](https://vitejs.dev/)
-- `node-todo`: A simple task manager template that uses Node.js and
+- `node-react-todo`: A simple task manager template that uses Node.js and
     [React](https://react.dev/). It demonstrates the use of the database for
     Create, Read, Update and Delete (CRUD) operations. It is built by
     [`vite`](https://vitejs.dev/)
@@ -95,7 +95,7 @@ template, you can execute:
 ```sh
 npm create @oracle/database-app -- \
     --name 'my-todo' \
-    --template 'node-todo'
+    --template 'node-react-todo'
 ```
 
 ## Contributing

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@oracle/create-database-app",
-  "version": "1.1.9",
+  "version": "1.1.10",
   "description": "Create an Oracle Database Application from a Template",
   "author": "Oracle Corporation",
   "keywords": [

